service/src/main/kotlin/no/nav/su/se/bakover/service/søknad/SøknadServiceImpl.kt

```diff
@@ -75,41 +105,45 @@ class SøknadServiceImpl(
             log.error("Ny søknad: Personen har et nyere fødselsnummer i PDL enn det som ble sendt inn. Bruker det nyeste fødselsnummeret istedet. Personoppslaget burde ha returnert det nyeste fødselsnummeret og bør sjekkes opp.")
         }
 
-        val (sakInfo: SakInfo, søknad: Søknad.Ny) = sakService.hentSakidOgSaksnummer(fnr).fold(
+        val soektSakstype = søknadInnhold.type()
+        // v1
+        val lol = sakService.hentSakidOgSaksnummer(fnr).fold(
             {
                 log.info("Ny søknad: Fant ikke sak for fødselsnummmer. Oppretter ny søknad og ny sak.")
-                val nySak = sakFactory.nySakMedNySøknad(
-                    fnr = fnr,
-                    søknadInnhold = søknadsinnholdMedNyesteFødselsnummer,
-                    innsendtAv = identBruker,
-                ).also {
-                    sakService.opprettSak(it)
-                }
-                val sakIdSaksnummerFnr = sakService.hentSakidOgSaksnummer(fnr)
-                    .getOrElse { throw RuntimeException("Feil ved henting av sak") }
-                Pair(sakIdSaksnummerFnr, nySak.søknad)
+                opprettNySøknad(fnr, søknadsinnholdMedNyesteFødselsnummer, identBruker)
```

Comment:
opprettNySøknad(fnr, søknadsinnholdMedNyesteFødselsnummer, identBruker).getOrElse {return it.left()}

Eller den leften du ønsker